lib/devise/models/database_authenticatable.rb

```diff
@@ -64,7 +64,7 @@ def clean_up_passwords
       # users to change relevant information like the e-mail without changing
       # their password). In case the password field is rejected, the confirmation
       # is also rejected as long as it is also blank.
-      def update_with_password(params, *options)
+      def update_with_password(params, **options)
```

Comment:
Can we add a test for this change? I could add it closest to this one https://github.com/plataformatec/devise/blob/master/test/models/database_authenticatable_test.rb#L155